.travis.yml

```diff
@@ -2,7 +2,11 @@ language: ruby
 script: rspec spec
 rvm:
   - 2.0.0
-  - 2.1.1
+  - 2.1.9
```

Comment:
I believe that we can drop anything below 2.3.7 (the version currently on MacOS), and we _should_ drop anything below 2.4.6 (latest 2.4 series) as everything below 2.4 is EOL.